src/utils/is.js

```diff
@@ -56,3 +56,20 @@ export const isAbortSignal = object => {
 		)
 	);
 };
+
+/**
+ * isDomainOrSubdomain reports whether sub is a subdomain (or exact match) of
```

Comment:
Should this use the public suffix list instead? https://publicsuffix.org/

Reply:
I don't think that would be necessary.